templates/index.html

```diff
@@ -7,9 +7,53 @@
   <script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
 
 <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+<link rel="stylesheet" type="text/css" href="/public/stylesheets.css">
```

Comment:
I think it would be better instead of replying every comment, you should add a commit with all the changes fix and notify us so we can review again the PR